viadot/flows/sap_rfc_to_adls.py

```diff
@@ -0,0 +1,118 @@
+import pandas as pd
+from typing import Any, Dict, List, Literal
+from prefect import Flow, task, unmapped
+
+from viadot.tasks import SAPRFCToDF
+from viadot.tasks import AzureDataLakeUpload
+from viadot.task_utils import df_to_csv
+
+download_sap_task = SAPRFCToDF()
+file_to_adls_task = AzureDataLakeUpload()
+
+
+@task
+def concat_dfs(dfs: List[pd.DataFrame]):
+    """
+    Task to combine list of data frames into one
+
+    Args:
+        dfs (List[pd.DataFrame]): List of dataframes to concat.
+    Returns:
+        full_df (pd.DataFrame()): Pandas dataframe containing all columns from dataframes from list.
+    """
+    full_df = pd.DataFrame()
+    for df in dfs:
+        full_df = pd.concat([full_df, df], axis=1)
+    return full_df
+
+
+class SAPRFCToADLS(Flow):
+    def __init__(
+        self,
+        name: str,
+        query_list: List[str],
+        sep: str = None,
+        func: str = "BBP_RFC_READ_TABLE",
+        sap_credentials: dict = None,
+        local_file_path: str = None,
+        file_sep: str = "\t",
+        if_exists: Literal["append", "replace", "skip"] = "replace",
+        adls_path: str = None,
+        overwrite: bool = False,
+        sp_credentials_secret: str = None,
+        vault_name: str = None,
+        *args: List[any],
+        **kwargs: Dict[str, Any],
+    ):
+        """
+        Flow for downloading data from SAP DataBase using the RFC protocol and uploading it to Azure Storage Explorer.
+
+        Note that only a very limited subset of SQL is supported:
+            - aliases
+            - where clauses combined using the AND operator
+            - limit & offset
+
+        Unsupported:
+            - aggregations
+            - joins
+            - subqueries
+            - etc.
+
+        Args:
+            name (str): The name of the flow.
+            query_list(List[str]) The list of queries to be executed with pyRFC.
+            sep(str, optional): Which separator to use when querying SAP. If not provided, multiple options are automatically tried.
```

Comment:
I think if you qualify one, you should qualify the other, so if you have `file_sep`, this should be eg. `query_sep`, `rfc_sep`, or similar.